components/_patterns/02-molecules/address-bar/address-bar.twig

```diff
@@ -0,0 +1,93 @@
+{#
+ # Variables:
+ # # Date Bar:
+ # address_bar__date_start - The start date/time.
+ # address_bar__date_end - The end date/time.
+ #
+ # # Location:
+ # address_bar__location - The name of the location.
+ # address_bar__address - The address of the location.
+ # address_bar__directions_url - The link to directions.
+ # address_bar__phone_number - The phone number of the location.
+ #
+ # # Hours:
+ # days - An array of days and hours
+ # - day - Day of the week.
+ # - hours - Hours open that day.
+ #
+ # # Accessibility:
+ # address_bar__accessibility - The accessibility level.
+ # accessibility_level__description - The description of the accessibility level.
+ #
+ # Amenities:
+ # location__amenities - An array of amenities
+ # - (Each amenity is an array item, as text)
+ #
+ # Secondary CTAs:
+ # address_bar__cta__secondary - An array of secondary links.
+ # - address_bar__cta__secondary__text - Secondary link text.
+ # - address_bar__cta__secondary__url - Secondary link url.
+ #
+ # Primary CTA:
+ # address_bar__cta__primary__text - Primary link text.
+ # address_bar__cta__primary__url - Primary link url.
```

Comment:
Can you add in the docs for `address_bar__link__alternate_text` here as well?

Reply:
yep